addons/web/static/src/scss/primary_variables.scss

```diff
@@ -186,3 +186,6 @@ $o-btn-secondary-hover-border: $o-gray-200 !default;
 $o-btn-light-bg: $o-white !default;
 $o-btn-light-border: $o-btn-light-bg !default;
 $o-btn-light-background-hover: $o-gray-100 !default;
+
+// Borders
+$hr-color: pink !default;
```

Comment:
Good job 👍 
But the exercise states that the custom color should only be applied in Community.
For now it applies to both.